Dockerfile

```diff
@@ -0,0 +1,87 @@
+FROM ubuntu:bionic
+WORKDIR /root
+
+# Install bitcoind
+RUN apt-get update \
+  && apt-get install -y software-properties-common \
+  && add-apt-repository ppa:bitcoin/bitcoin \
+  && apt-get update \
+  && apt-get install -y \
+    build-essential \
+    libtool \
+    autotools-dev \
+    automake \
+    pkg-config \
+    libssl-dev \
+    libevent-dev \
+    bsdmainutils \
+    python3 \
+    libboost-all-dev \
+    miniupnpc \
+    libzmq3-dev \
+    libdb4.8-dev \
+    libdb4.8++-dev \
+    git
+RUN git clone https://github.com/bitcoin/bitcoin.git \
+  && cd bitcoin \
+  && git checkout b641f60425674d737d77abd8c49929d953ea4154 \
+  && ./autogen.sh \
+  && ./configure \
+  && make \
+  && make install \
+  && cd /root \
+  && rm -rf bitcoin
+
+# Install lightning-integration
+RUN apt-get install -y \
+  python3 \
+  python3-pip \
+  libsecp256k1-dev \
+  && git clone https://github.com/cdecker/lightning-integration.git \
+  && pip3 install -r lightning-integration/requirements.txt
+
+WORKDIR /root/lightning-integration
```

Comment:
Setting workdir twice is not really needed if you instead provide a target clone directory. I'd rather have something like this:

```
git clone https://github.com/cdecker/lightning-integration.git /build
WORKDIR /build
```

Reply:
That's true!
I fixed it in https://github.com/cdecker/lightning-integration/pull/35/commits/1fee81ed9bcd62c8dad5ceb0f74088981c99bb49